### DISCUSSION TITLE
is_emergency_motion

### DISCUSSION THREAD

**Discussion by Naoki-Hiraoka** [YutaKojio/hrpsys-base#21] [Q&A] [answered]
is_emergency_motionはstabliezerの該当箇所をコメントインすると使えるそうですが、なにをする機能なのでしょうか?
https://github.com/YutaKojio/hrpsys-base/blob/c40cffe9818d33cae39ed961b99f5562c907c828/rtc/AutoBalancer/Stabilizer.cpp#L1370-L1375

**Comment by YutaKojio** [comment, marked as answer]
https://drive.google.com/file/d/1MQ4hQQ9ykcdtHHqoxJJltHnLaHdl74tS/view?usp=sharing
のように転倒動作に移る用です．
直立状態で押された時の転倒回避の段階として外乱の大きさに応じて
Emergency stop(動作を止める) -> 踏み出す -> 転倒動作
の順に行われます．

転倒動作は
https://github.com/YutaKojio/hrpsys-base/blob/5aab7f9809b1ef76dd62bf323e522834d84097be/rtc/EmergencyStopper/EmergencyStopper.cpp#L327
にハードコードです
